apple/Handlers/RNNativeViewHandler.mm

```diff
@@ -39,20 +39,27 @@ - (id)initWithGestureHandler:(RNGestureHandler *)gestureHandler
 
 - (void)touchesBegan:(NSSet<RNGHUITouch *> *)touches withEvent:(UIEvent *)event
 {
+  [self setStateIfScrollView:UIGestureRecognizerStatePossible];
```

Comment:
I don't think this is needed. `UIGestureRecognizerStatePossible` should be the starting state by default.

Reply:
Yes, I've just checked and it is the starting state.